janus_core/calculations/phonons.py

```diff
@@ -44,8 +44,11 @@ class Phonons:  # pylint: disable=too-many-instance-attributes
         Whether to plot various graphs as band stuctures, dos/pdos in svg.
         Default is False.
     symmetrize : bool
-        Whether to symmetrize force constants after cauclation.
+        Whether to symmetrize force constants after calculation.
         Default is False.
+    write_full : bool
+        Whether to maximize information written in various output files.
```

Comment:
If that's what it's doing, then I think `write_full_bands` or `write_full_info` would be a better name and specify which data are affected nad possibly how.

Reply:
initially I was looking at write_full_info but wanted to keep it shorter. plan is to extend it to other things.